transcendence-app/src/chat/chat.controller.ts

```diff
@@ -55,7 +55,7 @@ export class ChatController {
     return chat;
   }
 
-  @Post(':chatId')
+  @Post(':chatId/members')
```

Comment:
We may want to have another endpoint, `/<chatId>/members/<userId>` so we can add other users apart from current? (I am thinking in an adming adding users to a chatroom)